lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -443,3 +443,17 @@ def test_multi_font_type42():
 
     fig = plt.figure()
     fig.text(0.15, 0.475, "There are 几个汉字 in between!")
+
+
+def test_otf_font_smoke():
```

Comment:
This test appears to be skipped on CI, which we probably don't want, can it be changed to use a different font that is already on CI at least?